Lombiq.Hosting.MediaTheme.Deployer/Program.cs

```diff
@@ -69,9 +69,10 @@ internal static class Program
 
     private static void HandleParseError(IEnumerable<Error> errors)
     {
-        if (errors.Any())
+        var errorsList = errors.ToList();
```

Comment:
This won't be a `List<T>` behind the scenes.

Reply:
There isn't much point to making performance optimizations in error handling anyway, since that kind of stuff is exceptional.

Reply:
Indeed.